Configurations.md

```diff
@@ -1007,7 +1007,40 @@ use lorem;
 use sit;
 ```
 
-See also [`reorder_imported_names`](#reorder_imported_names).
+See also [`reorder_imported_names`](#reorder_imported_names), [`reorder_imports_in_group`](#reorder_imports_in_group).
+
+## `reorder_imports_in_group`
+
+Reorder import statements in group
```

Comment:
IMHO it would be better if there is an instruction to use `reorder_imports_in_group` with `reorder_imports`, since it seems that just writing `reorder_imports_in_group = true` to rustfmt.toml does not take effect.